WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderFollowButton.java

```diff
@@ -0,0 +1,78 @@
+package org.wordpress.android.ui.reader.views;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.animation.AnimatorSet;
+import android.animation.ObjectAnimator;
+import android.animation.ValueAnimator;
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.animation.AccelerateDecelerateInterpolator;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+
+/**
+ * Follow button used throughout the reader
+ */
+public class ReaderFollowButton extends LinearLayout {
+    private TextView mTextFollow;
+    private boolean mIsFollowed;
+
+    public ReaderFollowButton(Context context){
+        super(context);
+        initView(context);
+    }
+
+    public ReaderFollowButton(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        initView(context);
+    }
+
+    public ReaderFollowButton(Context context, AttributeSet attrs, int defStyle) {
+        super(context, attrs, defStyle);
+        initView(context);
+    }
+
+    private void initView(Context context) {
+        inflate(context, R.layout.reader_follow_button, this);
+        mTextFollow = (TextView) findViewById(R.id.text_follow_button);
+    }
+
+    private void updateFollowText() {
+        mTextFollow.setSelected(mIsFollowed);
+        mTextFollow.setText(mIsFollowed ? R.string.reader_btn_unfollow : R.string.reader_btn_follow);
+    }
+
+    public void setIsFollowed(boolean isFollowed, boolean animateChanges) {
```

Comment:
You're right - that's bad style on my part, and I even made a similar comment on a recent PR from Danilo. I'll correct this.